generalTransform.h

```diff
@@ -31,6 +31,15 @@ class GeneralTransform {
   Tristate::Tristate isInPlace;  // otherwise out-of-place
   Tristate::Tristate isReal;     // otherwise C2C
 
+  int originalX;
+  int originalY;
+  int originalZ;
+
+  int kernelInvocationX;
+  int kernelInvocationY;
+  int kernelInvocationZ;
+
+  bool swapped2D;
```

Comment:
Do we need to keep this info? It should be the `transformGenerator` that generates swapped transformations, and the rest of the code should not care about this information. Correct?